core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -104,12 +94,23 @@ object TestUtils extends Logging {
    * Create a temporary relative directory
    */
   def tempRelativeDir(parent: String): File = {
-    val f = new File(parent, "kafka-" + random.nextInt(1000000))
-    f.mkdirs()
+    new File(parent).mkdirs()
```

Comment:
@ijuma @fpj I had tried that the first time, but remember seeing test failures because some tests use this directory as the partition directory and require the format `string-int`. Could create parent as a temp directory, but that is changing the semantics of the method and needs reviewing all invocations. Hence the simpler change that is contained in this method retaining the original semantics.